tests/upgrade/basic/task.yaml

```diff
@@ -13,7 +13,7 @@ execute: |
     . "$TESTSLIB/apt.sh"
 
     echo Install previous version...
-    dpkg -l snapd snap-confine || true
+    dpkg -l snapd snap-wrap || true
```

Comment:
This was an accidental sed, as is refers to package names.  Though it's also possible this is obsolete now, and all you care about is snapd?